packages/nuxt/src/pages/runtime/plugins/router.ts

```diff
@@ -278,4 +278,9 @@ const plugin: Plugin<{ router: Router }> = defineNuxtPlugin({
   },
 })
 
+const ENCODED_TRAILING_SLASH_RE = /%2F$/
+function isSamePath (initialURL: string, path: string) {
+  return withoutTrailingSlash(decode(initialURL).replace(ENCODED_TRAILING_SLASH_RE, '/')) === withoutTrailingSlash(decode(path).replace(ENCODED_TRAILING_SLASH_RE, '/'))
+}
```

Comment:
Is this something we should consider fixing/handling in `ufo` too?

Reply:
What do you think @pi0?